bin/include/jvmdefaults.bat

```diff
@@ -42,34 +42,13 @@ if %java_version% GEQ 9 if %java_version% LSS 11 (
     %current_value%
 )
 
-if %java_version% GEQ 11 if %java_version% LSS 14 (
+if %java_version% GEQ 11 (
     set value= ^
     --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED ^
     --add-exports=java.base/sun.nio.ch=ALL-UNNAMED ^
     --add-exports=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED ^
     --add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED ^
     --add-exports=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED ^
-    --add-opens=jdk.management/com.sun.management.internal=ALL-UNNAMED ^
-    --illegal-access=permit ^
-    %current_value%
-)
-
-if %java_version% GEQ 14 if %java_version% LSS 15 (
-    set value= ^
-    --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED ^
-    --add-exports=java.base/sun.nio.ch=ALL-UNNAMED ^
-    --add-exports=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED ^
-    --add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED ^
-    --add-exports=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED ^
-    --add-opens=java.base/jdk.internal.access=ALL-UNNAMED ^
-    --add-opens=jdk.management/com.sun.management.internal=ALL-UNNAMED ^
-    --illegal-access=permit ^
-    %current_value%
-)
-
-if %java_version% GEQ 15 (
-    set value= ^
-    --add-opens=java.base/jdk.internal.access=ALL-UNNAMED ^
     --add-opens=java.base/jdk.internal.misc=ALL-UNNAMED ^
```

Comment:
Does it make sense to have both options for the same module/package?
```
if %java_version% GEQ 11 (
    set value= ^
    --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED ^
    ...
    --add-opens=java.base/jdk.internal.misc=ALL-UNNAMED ^
    ...
```

Reply:
Hi @sk0x50 ,

In my tests across various Java versions, the presence of both options didn't impact the execution outcome or the suppression of illegal reflective access warnings. Given these two options serve different purposes in terms of access level and scope, I believe it's appropriate to keep both. Please let me know if you prefer a different approach. 

Additionally, the issue with inclusion of the `build-classpath.sh` reference is still pending your confirmation. 

Thanks.